### PR TITLE
added patrick mega course & a few others

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
Added a few more courses, including the mega 32 hour long course in javascript!

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/6481

### CODE REVIEW

File: src/data/externalTutorials.json [hunk 1]
```diff
@@ -30,6 +72,108 @@
...
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "foundry", "bash", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "19",
+    "lang": "en",
+    "publishDate": "03/28/2022"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=pdsYCkUWrgQ",
+    "title": "How to Connect your Smart Contracts to Metamask | Full Stack Web3",
```
Comment (minimalsm) [inline src/data/externalTutorials.json:89]:
```suggestion
    "title": "How to Connect your Smart Contracts to Metamask",
```

File: src/data/externalTutorials.json [hunk 2]
```diff
@@ -30,6 +72,108 @@
...
+    ],
+    "skillLevel": "beginner",
+    "timeToRead": "70",
+    "lang": "en",
+    "publishDate": "02/11/2022"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=pdsYCkUWrgQ",
+    "title": "How to build an on-chain DAO",
+    "description": "Using Compound and Openzeppelin as a basis, we build a 100% on-chain DAO using an ERC20 governance token for votes.  ",
```
Comment (minimalsm) [inline src/data/externalTutorials.json:116]:
```suggestion
    "description": "Using Compound and Openzeppelin as a basis, we build a 100% on-chain DAO using an ERC20 governance token for votes.",
```

File: src/data/externalTutorials.json [hunk 3]
```diff
@@ -30,6 +72,108 @@
     "lang": "en",
     "publishDate": "09/09/2021"
   },
+  {
+    "url": "https://www.youtube.com/watch?v=fNMfMxGxeag",
+    "title": "Introduction to Foundry",
+    "description": "We build a minimal foundry project using a staking application. Showing you how to work with foundry.  ",
```
Comment (minimalsm) [inline src/data/externalTutorials.json:78]:
```suggestion
    "description": "We build a minimal Foundry project using a staking application to show you how to work with Foundry.",
```

File: src/data/externalTutorials.json [hunk 4]
```diff
@@ -30,6 +72,108 @@
...
+    "tags": ["solidity", "foundry", "bash", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "19",
+    "lang": "en",
+    "publishDate": "03/28/2022"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=pdsYCkUWrgQ",
+    "title": "How to Connect your Smart Contracts to Metamask | Full Stack Web3",
+    "description": "We learn exactly how web3 / blockchain / smart contract applications work in the front end using HTML and Javascript. We then go through 6 different ways you can connect your Metamask, Phantom, or other blockchain wallet address to your front end. We’ll look at popular Nextjs / React packages to make your development lifecycle 100 times easier. ",
```
Comment (minimalsm) [inline src/data/externalTutorials.json:90]:
```suggestion
    "description": "We learn exactly how web3 / blockchain / smart contract applications work in the front end using HTML and Javascript. We then go through 6 different ways you can connect your Metamask, Phantom, or other blockchain wallet address to your front end. We’ll look at popular Nextjs / React packages to make your development lifecycle 100 times easier.",
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks for the PR @PatrickAlphaC, just a few comments to remove some deadspace and remove some extra newly added tags (we're overflowing in tags and need to explore a new solution 😅)

**Review by minimalsm — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
Love it!

Thanks for your work here!

**Pull request merged**